ImperatorToCK3/Source/Imperator/Families/Family.h

```diff
@@ -2,33 +2,41 @@
 #define IMPERATOR_FAMILY_H
 
 
+
 #include <string>
+#include <memory>
+#include <vector>
+
 
 
-namespace Imperator
-{
-class Family
-{
+namespace Imperator {
+class Character;
+class Family {
   public:
 	class Factory;
 	Family() = default;
 
+	void linkMember(const std::shared_ptr<Character>& memberPtr);
+
+	[[nodiscard]] auto getID() const { return ID; }
+	[[nodiscard]] const auto& getKey() const { return key; }
 	[[nodiscard]] const auto& getCulture() const { return culture; }
 	[[nodiscard]] const auto& getPrestige() const { return prestige; }
 	[[nodiscard]] const auto& getPrestigeRatio() const { return prestigeRatio; }
-	[[nodiscard]] const auto& getKey() const { return key; }
-	[[nodiscard]] const auto& getIsMinor() const { return isMinor; }
+	[[nodiscard]] const auto& getMembers() const { return members; }
+	[[nodiscard]] const auto& isMinor() const { return minor; }
 
-	[[nodiscard]] auto getID() const { return ID; }
 
   private:
 	unsigned long long ID = 0;
+	std::string key;
 	std::string culture;
 	double prestige = 0;
 	double prestigeRatio = 0;
-	std::string key;
-	bool isMinor = false;
+	std::vector<std::pair<unsigned long long, std::shared_ptr<Character>>> members;
```

Comment:
Any reason to not use a map?

Reply:
Yes, the first member seems to be the head.